game/src/core/level/utils/LevelUtils.java

```diff
@@ -0,0 +1,140 @@
+package core.level.utils;
+
+import com.badlogic.gdx.ai.pfa.GraphPath;
+
+import contrib.utils.components.ai.AITools;
+
+import core.Entity;
+import core.Game;
+import core.components.PositionComponent;
+import core.level.Tile;
+import core.utils.Point;
+import core.utils.components.MissingComponentException;
+
+import java.util.Optional;
+
+public class LevelUtils {
+    /**
+     * Finds the path from the given point to another given point.
+     *
+     * <p>Throws an IllegalArgumentException if the tile at 'from' or 'to' is non-accessible.
+     *
+     * @param from The start point.
+     * @param to The end point.
+     * @return Path from the start point to the end point.
+     */
+    public static GraphPath<Tile> calculatePath(final Point from, final Point to) {
+        return calculatePath(from.toCoordinate(), to.toCoordinate());
+    }
+
+    /**
+     * Finds the path from the given coordinate to another given coordinate.
+     *
+     * <p>Throws an IllegalArgumentException if the tile at the start or end is non-accessible.
+     *
+     * @param from The start coordinate.
+     * @param to The end coordinate.
+     * @return Path from the start coordinate to the end coordinate.
+     */
+    public static GraphPath<Tile> calculatePath(final Coordinate from, final Coordinate to) {
+        return Game.findPath(Game.tileAT(from), Game.tileAT(to));
+    }
+
+    /**
+     * Finds the path to a random (accessible) tile in the given radius, starting from the given
+     * point.
+     *
+     * <p>If there is no accessible tile in the range, the path will be calculated from the given
+     * start point to the given start point.
+     *
+     * <p>Throws an IllegalArgumentException if the tile at the start point is non-accessible.
+     *
+     * @param point The start point.
+     * @param radius Radius in which the tiles are to be considered.
+     * @return Path from the center point to the randomly selected tile.
+     */
+    public static GraphPath<Tile> calculatePathToRandomTileInRange(
+            final Point point, final float radius) {
+        Coordinate newPosition =
+                AITools.randomAccessibleTileCoordinateInRange(point, radius)
```

Comment:
Das ist glaub ein ganz guter Indikator, dass das auch rüber muss.

Hätte gesagt:
`AITools` darf `LevelUtils` verwenden, da diese in `core` liegt, umgedreht aber nicht

=> Alles `tileInRange` o.ä mit nach LevelUtils

Und wenn wir schonmal hier sind: 
Nenn `AITools` mal in `AIUtils` um